src/js/utils/validator.ts

```diff
@@ -6,7 +6,7 @@ const protocol = window.location.protocol;
  * @param {any} item - The variable to test.
  * @returns {boolean} Is the value of `item` null, undefined or an empty string?
  */
```

Comment:
Should we keep these JSDOC comments?

Reply:
i'm fine with punting this as long as the comments match the type definitions